src/object.d

```diff
@@ -4236,10 +4236,18 @@ Params:
 
 Return: calculated hash value of $(D arg)
 */
-size_t hashOf(T)(auto ref T arg, size_t seed = 0)
+version (D_Ddoc)
 {
-    import core.internal.hash;
-    return core.internal.hash.hashOf(arg, seed);
+    // This lets DDoc produce better documentation.
+    size_t hashOf(T)(auto ref T arg, size_t seed = 0)
+    {
+        static import core.internal.hash;
+        return core.internal.hash.hashOf(arg, seed);
```

Comment:
Do we even need a body if it's for ddoc only? An empty body should work too?

Reply:
Should work.

Reply:
When I replace it with `size_t hashOf(T)(auto ref T arg, size_t seed = 0);` I get an error:
```
../dmd/generated/osx/release/64/dmd -conf= -c -w -o- -Isrc -Iimport -version=CoreDdoc -Dfdoc/object.html project.ddoc  src/object.d
src/object.d(1605): Error: function object.hashOf!(void delegate()).hashOf is not nothrow
src/object.d(1603): Error: nothrow function object.TypeInfo_Delegate.getHash may throw
```